pkg/crc/machine/machine.go

```diff
@@ -513,8 +513,18 @@ func Status(statusConfig ClusterStatusConfig) (ClusterStatusResult, error) {
 			return *result, errors.New(err.Error())
 		}
 		if operatorsRunning {
-			// TODO:get openshift version as well and add to status
-			openshiftStatus = fmt.Sprintf("Running (v4.x)")
+			var openshiftVersion string
+			_, crcBundleMetadata, err := getBundleMetadataFromDriver(host.Driver)
+			if err != nil {
+				logging.Debugf("Failed to load bundle metadata: %s", err.Error())
+			}
+			if crcBundleMetadata != nil {
+				openshiftVersion = crcBundleMetadata.GetOpenshiftVersion()
+			}
+			if openshiftVersion == "" {
+				openshiftVersion = "4.x"
+			}
```

Comment:
@cfergeau I would do something like following.
```golang
                      openshiftVersion := "4.x"
			_, crcBundleMetadata, err := getBundleMetadataFromDriver(host.Driver)
			if err != nil {
				logging.Debugf("Failed to load bundle metadata: %s", err.Error())
			}
			if crcBundleMetadata.GetOpenshiftVersion() != "" {
				openshiftVersion = crcBundleMetadata.GetOpenshiftVersion()
			}
```

Which condition do you think when there is no error and `crcBundleMetadata` is also nil?